easybuild/easyconfigs/p/protobuf/protobuf-3.21.9-foss-2021a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'protobuf'
+version = '3.21.9'
+
+homepage = 'https://github.com/protocolbuffers/protobuf'
+description = """Protocol Buffers (a.k.a., protobuf) are Google's 
+language-neutral, platform-neutral, extensible mechanism for 
+serializing structured data."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
```

Comment:
@guacke do you have a good reason to use the `foss` toolchain instead of `GCCcore`, which is used for all the recent `protobuf` easyconfigs?
if not, please change to `GCCcore`.

Reply:
+1

@guacke please move to `GCCcore/10.3.0` (or close this and use the existing `protobuf-3.17.3-GCCcore-10.3.0.eb` if that's recent enough for #16738)